api/travel.php

```diff
@@ -52,9 +52,14 @@
 
             $success = $TravelManager->updateFilter($filter, $filter_value);
             $TravelAPIResponse->response = TravelApiPresenter::travelActionResponse($success, $player, $TravelManager);
-
             break;
 
+        case 'AttackPlayer':
+            $target_user_id = $system->db->clean($_POST['target']);
+
+            $success = $TravelManager->attackPlayer($target_user_id);
```

Comment:
This isn't actually the user's ID, this is the user's "attack ID" which is a random hash regenerated regularly to prevent bookmarking a specific user's attack link
```suggestion
            $target_attack_id = $system->db->clean($_POST['target_attack_id']);

            $success = $TravelManager->attackPlayer($target_attack_id);
```